source/linux/Foundational_Components/Power_Management/pm_low_power_modes.rst

```diff
@@ -290,6 +291,126 @@ system and it will go through a normal Linux boot process.
+I/O Only Plus DDR
+*****************
+
+.. ifconfig:: CONFIG_part_variant in ('AM62X')
+
+   This mode is not applicable for AM62X.
+
+.. ifconfig:: CONFIG_part_variant in ('AM62AX' , 'AM62PX')
+
+   This mode is similar to Partial I/O mode, with the major distinction being
+   that the DDR memory is kept in self refresh to save context. All the processor
+   power supplies are turned off except the LVCMOS I/O power supply while keeping
+   DDR in self-refresh.
+   The user can do system power state transitions, including power supply control,
+   by a single interface signal (PMIC_LPM_EN signal) with PMIC register programming.
+
+   The benefits of using I/O Only plus DDR in embedded devices:
+
+   #. Lowest power consumption: I/O Only Plus DDR mode can save a significant amount of power, especially in battery-powered
+      devices that are mostly idle or low activity most of the time with the full context being saved.
+   #. Better efficiency: I/O Only Plus DDR mode can help to improve the efficiency of embedded devices by
+      reducing the amount of time that the processor is idle. This is because the processor can
+      be kept in a low-power state when it is not needed.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+      .. important:: Jumper J9 should be connected on SK to enable system to enter I/O Only plus DDR mode.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+      .. important:: Jumper J12 should be connected on SK to enable system to enter I/O Only plus DDR mode.
+
+   The wakeup sources that can be used to wake the system from I/O Only Plus DDR are
+   mcu_uart0, mcu_mcan0, mcu_mcan1 and wkup_uart0. After Linux boots, direct register
+   writes can be used to enable wakeup.
+
+   .. rubric:: Following commands can be used to enable wakeup from mcu_mcan0, mcu_mcan1,
+               mcu_uart0 and wkup_uart0 respectively.
+
+   .. code-block:: console
+
+      root@<machine>:~# devmem2 0x4084038 0x20050000
+      root@<machine>:~# devmem2 0x4084040 0x20050000
+      root@<machine>:~# devmem2 0x4084014 0x20050000
+      root@<machine>:~# devmem2 0x4084024 0x20050000
+
+   .. note::
+
+      Atleast one of the wakeup sources listed above must be enabled to wakeup from I/O Only Plus DDR mode.
+
+   .. rubric:: To enter I/O Only Plus DDR mode, first disable wakeup from RTC, USB0 and USB1 as these wakeup
+               sources are not supported for this mode.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+      .. code-block:: console
+
+         root@am62axx-evm:~# echo disabled > /sys/class/rtc/rtc0/device/power/wakeup
+         root@am62axx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f900000.dwc3-usb/power/wakeup
+         root@am62axx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f910000.dwc3-usb/power/wakeup
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+      .. code-block:: console
+
+         root@<machine>:~# echo disabled > /sys/class/rtc/rtc0/device/power/wakeup
```

Comment:
Better to have a single convention for the entire section.

- If it's a common section, use \<machine\>
- if it's under ifconfig, use the actual yocto machine name

Since it's under CONFIG_part_variant in ('AM62PX')
```suggestion
         root@<am62pxx-evm>:~# echo disabled > /sys/class/rtc/rtc0/device/power/wakeup
```

Reply:
I concur. Would you mind adding that to the contribution guide @cshilwant ?

Reply:
Updated. Thanks for pointing this out